native/index.js

```diff
@@ -0,0 +1,10 @@
+// this is needed to support socket.io
+window.navigator.userAgent = 'react-native';
+
+const { AppRegistry } = require('react-native');
```

Comment:
Could we move this to the top? Or is the import depending on userAgent to be set?